zaza/openstack/charm_tests/ceph/tests.py

```diff
@@ -685,6 +691,15 @@ def multisite(self):
         except KeyError:
             return False
 
+    @property
+    def cloud_sync(self):
+        """Determine whether Ceph cloud sync is used."""
```

Comment:
NIT: I think this doc string is not 100% accurate. Maybe something like, `Determine whether ceph cloud sync application is present` is clearer ?

Reply:
Yep, I agree. I [changed this](https://github.com/openstack-charmers/zaza-openstack-tests/compare/a474f61d4d3ec30b73091276033482e0e684ef8b..59876270cfe372bdf744b6eed9557d1763a9a251).